src/main/java/org/ballerinalang/command/cmd/ListCommand.java

```diff
@@ -37,13 +36,13 @@
 import java.util.Arrays;
 import java.util.List;
 
+import static org.ballerinalang.command.util.OSUtils.getBallerinaDistListFilePath;
```

Comment:
Instead of importing like this can't you just call `OSUtils.getBallerinaDistListFilePath()` in line 170?